modules/WebPublishPlugin/src/main/java/net/clementlevallois/web/publish/plugin/controller/JPanelWebExport.java

```diff
@@ -0,0 +1,612 @@
+/*
+ * author: Clément Levallois
+ */
+package net.clementlevallois.web.publish.plugin.controller;
+
+import com.google.gson.JsonElement;
+import com.google.gson.JsonObject;
+import com.google.gson.JsonParser;
+import java.awt.Color;
+import java.io.IOException;
+import java.net.URI;
+import java.net.http.HttpClient;
+import java.net.http.HttpRequest;
+import java.net.http.HttpResponse;
+import java.util.ResourceBundle;
+import java.util.UUID;
+import java.util.prefs.Preferences;
+import javax.swing.JLabel;
+
+import javax.swing.JTextField;
+import javax.swing.SwingWorker;
+import static net.clementlevallois.web.publish.plugin.controller.GlobalConfigParams.*;
+import org.openide.DialogDescriptor;
+import org.openide.DialogDisplayer;
+import org.openide.NotifyDescriptor;
+import org.openide.util.Exceptions;
+import org.openide.util.NbBundle;
+import org.openide.util.NbPreferences;
+
+/**
+ *
+ * @author LEVALLOIS
+ *
+ * The logic of this plugin follows the logic laid out here:
+ * https://github.com/gephi/gephi-plugins/issues/262#issuecomment-1231627948
+ *
+ */
+public class JPanelWebExport extends javax.swing.JPanel {
```

Comment:
I wouldn't start a class name with JPanel. Rather start with the class purpose and finish by panel or JPanel.